server/services/models/appointmentsDTO.js

```diff
@@ -1,6 +1,6 @@
 const Joi = require("joi");
 const Moment = require('moment');
-const MONTHS = require('../../utils/months.constant');
+const FILTERDATE = require('../../utils/getDate.constants');
```

Comment:
If you only use filterDate function from this module, you can import it like
```suggestion
const { filterDate } = require('../../utils/getDate.constants');
```